run/types/DeviceWrapper.ts

```diff
@@ -1753,6 +1772,68 @@ export class DeviceWrapper {
     return;
   }
 
+  public async checkModalStrings(
+    expectedStringHeading: TokenString<LocalizerDictionary>,
+    expectedStringDescription: TokenString<LocalizerDictionary>,
+    oldModalAndroid?: boolean
+    // args?: { [key: string]: string | number }
+  ) {
+    // Check modal heading is correct
+    function replaceBrWithCRLF(input: string): string {
+      // return input.replace(/<br\s*\/?>/gi, '\nCR LF');
+      return input.replace(/\n/gi, '');
+    }
```

Comment:
the name is misleading, maybe rename it to `removeNewLines`
also, isn't playwright supposed to do those checks for you?

Reply:
Playwright currently does not do these checks, so had to create my own function. Will rename.